flow/flow.go

```diff
@@ -81,6 +80,14 @@ type RawPackets struct {
 	RawPackets []*RawPacket
 }
 
+// FlowOpts describes options that can be used to process flows
+type FlowOpts struct {
+	ParentUUID string
```

Comment:
why putting parentUUID / L2 and L3 as flow option ? It's not flow options
but may be factorize as FlowUUIDs struct

Reply:
I thought them as options as not always provided but yes we can use another struct. I'll do that